easybuild/tools/toolchain/toolchain.py

```diff
@@ -702,7 +703,7 @@ def prepare(self, onlymod=None, silent=False, loadmod=True, rpath_filter_dirs=No
 
         if build_option('rpath'):
             if self.options.get('rpath', True):
-                self.prepare_rpath_wrappers()
+                self.prepare_rpath_wrappers(rpath_filter_dirs, rpath_include_dirs)
```

Comment:
Wait, we were not passing down `rpath_filter_dirs` here?

That's a bug, right, which wasn't captured by the tests?

Reply:
Uhm, it seems so. I just passed it as well, because I wanted to pass `rpath_include_dirs` (which I guess I could've passed as keyword argument, still learning python..). The test in `test_toolchain_prepare_rpath` sets the build_option `rpath_filter`, while `prepare_rpath_wrappers` combines the build_option `rpath_filter` AND the optional argument `rpath_filter_dirs`. The last one is not tested (and apparently not yet used).

Reply:
OK, we should make sure the tests check the usage... I'll take a look at that.

Reply:
Enhanced test to catch this (now fixed) bug is included in updates in https://github.com/sara-nl/easybuild-framework/pull/1